compiler-rt/lib/fuzzer/FuzzerUtilFuchsia.cpp

```diff
@@ -607,7 +607,13 @@ size_t PageSize() {
 }
 
 void SetThreadName(std::thread &thread, const std::string &name) {
-  // TODO ?
+  if (name.size() > 31)
+    name.resize(31);
+  zx_status_t s;
+  if ((s = zx_object_set_property(thread.native_handle(), ZX_PROP_NAME,
+                                  name.c_str(), name.size())) != ZX_OK)
```

Comment:
This could be potentially simplified to:
```
if (zx_status_t status = zx_object_set_property(thread.native_handle(), ZX_PROP_NAME, name.c_str(), name.size()); status != ZX_OK)
```